modules/status/sound_bsd.go

```diff
@@ -0,0 +1,76 @@
+// +build openbsd freebsd netbsd
+
+package status
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	wmtheme "fyne.io/fynedesk/theme"
+)
+
+var volume int
+
+// Destroy destroys the module
```

Comment:
"Destroy destroys" seems odd.  Perhaps Destroy removes the module?

Reply:
This was old code refactored, will update